samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -86,6 +86,35 @@ The DTM sample supports all four PHYs specified in DTM, but not all devices supp
    * - LE Coded S=2
      - Yes
 
+Supported Direction finding modes
+=================================
+
+The DTM sample supports all Direction Finding modes specified in DTM, but not all device support the Direction Finding.
```

Comment:
```suggestion
The DTM sample supports all Direction Finding modes specified in DTM, but not all devices support the Direction Finding.
```

Reply:
We claim to support only nRF53 currently. I wonder if the second part of the sentence is required

Reply:
Ok I removed this sentence